model/riscv_sys_regs.sail

```diff
@@ -108,6 +108,14 @@ val sys_enable_vext = {c: "sys_enable_vext", ocaml: "Platform.enable_vext", _: "
 /* whether misa.b was enabled at boot */
 val sys_enable_bext = {c: "sys_enable_bext", ocaml: "Platform.enable_bext", _: "sys_enable_bext"} : unit -> bool
 
+// CBO extensions. Zicbop cannot be enabled/disabled because it has no effect
+// at all on this model.
+// TODO: Add command line flags.
```

Comment:
IIRC one of these conflicts with the current CHERI-RISC-V encoding for LC (and the community's historic unofficial consensus for RV128's LQ, which is where that came from), so sail-cheri-riscv needs this to be disableable in sail-riscv without code modification to riscv_sys_regs.sail.

Reply:
Ah yeah, you're right. `cbo.zero` is

```
mapping clause encdec = RISCV_ZICBOZ(rs1)                if haveZicboz()
  <-> 0b000000000100 @ rs1 @ 0b010 @ 0b00000 @ 0b0001111 if haveZicboz()
```

And that used to conflict with `lc`, which was this (I think `cd != zreg` was our temporary workaround):

```
mapping clause encdec = LoadCapImm(cd, rs1, offset) if haveXcheri() & sizeof(xlen) == 64 & cd != zreg  /* lc */
   <-> offset @ rs1 @ 0b010 @ cd @ 0b0001111 if haveXcheri() & sizeof(xlen) == 64 & cd != zreg
```

However that was [changed](https://riscv.github.io/riscv-cheri/#LC) so it's now this:

```
mapping clause encdec = LoadCapImm(cd, rs1, offset) if haveXcheri() & sizeof(xlen) == 64  /* lc */
  <-> offset @ rs1 @ 0b100 @ cd @ 0b0001111         if haveXcheri() & sizeof(xlen) == 64
```

Reply:
Yes but it's going to take a while to shift everything over to that...

Reply:
Ok I'll add command line flags.

Reply:
Done